### ISSUE TITLE
Persistent login TypeError: private key should be a Buffer

### ISSUE THREAD

**Issue by tinybike** [AugurProject/augur#724]
I'm getting the following error when I use a persistent login and try to refresh / browse to the UI:
```
assert.js 14: Uncaught TypeError: private key should be a Buffer
```
If the private key is converted to a string for localStorage, it should be converted back to a Buffer before use: `new Buffer(privateKey, 'hex')` should do it (assuming it's a hex string).

**Comment by tinybike** [comment]
Actually, it looks like the relevant method is in augur.js -- I'll fix.

**Comment by tinybike** [comment]
Fixed in 487b5bb2f878bc59abdc940d5d07cbca0fe75f7e and AugurProject/augur.js@6886486fe24394723d9555889c6ea438c479ce04
